src/io/flutter/sdk/FlutterCreateAdditionalSettings.java

```diff
@@ -100,6 +113,11 @@ public void setKotlin(boolean value) {
       args.add(org);
     }
 
+    if (!StringUtil.isEmptyOrSpaces(project)) {
+      args.add("--project-name");
```

Comment:
We should verify `--project-name` vs `--package-name`.

Reply:
Looks like `--project-name`; this was added to flutter ~17 months ago (https://github.com/flutter/flutter/commit/b95b67a66b64c94f9bbe1c011bbfddb593a64429).